src/fcntl.rs

```diff
@@ -164,6 +167,26 @@ pub fn openat<P: ?Sized + NixPath>(dirfd: RawFd, path: &P, oflag: OFlag, mode: M
     Errno::result(fd)
 }
 
+pub fn rename<P1: ?Sized + NixPath, P2: ?Sized + NixPath>(old: &P1, new: &P2) -> Result<()> {
+    let res = try!(try!(old.with_nix_path(|old_cstr| {
+        new.with_nix_path(|new_cstr| {
+            unsafe { libc::rename(old_cstr.as_ptr(), new_cstr.as_ptr()) }
+        })
+    })));
+    Errno::result(res).map(drop)
+}
+
+pub fn renameat<P1: ?Sized + NixPath, P2: ?Sized + NixPath>(old_dirfd: RawFd, old_path: &P1,
+                                                            new_dirfd: RawFd, new_path: &P2)
+                                                            -> Result<()> {
+    let res = try!(try!(old_path.with_nix_path(|old_cstr| {
```

Comment:
`?` is preferred over `try!`.  It's supported by Nix's MSRV.

Reply:
Done.